### ISSUE TITLE
when (if ever) should we update `Cargo.lock`?

### ISSUE THREAD

**Issue by ctb** [sourmash-bio/sourmash#2352]
Over the last few days, I just merged a bunch of Rust dependabot updates - e.g. https://github.com/sourmash-bio/sourmash/pull/2349. Should we be updating `Cargo.lock` as part of that, or as part of every release, or ...?

**Comment by luizirber** [comment]
Dependabot updates the Cargo.lock file too, see https://github.com/sourmash-bio/sourmash/pull/2349/files#diff-13ee4b2252c9e516a0547f2891aa2105c3ca71c6d7a1e682c69be97998dfc87e

**Comment by ctb** [comment]
yes, I know, but I've been having to resolve conflicts manually upon occasion.

seems like not a bad idea to periodically rebuild it from scratch, yah? or should I "resolve conflicts" that way in general? :)
